spec.md

```diff
@@ -293,7 +294,7 @@ Any closing brace MUST NOT be followed by any comment or statement on the
 same line.
 
 When instantiating a new class, parentheses MUST always be present even when
-there are no arguments passed to the constructor.
+there are no arguments passed to the constructor.  For example:
```

Comment:
Can you update these to all use a single space or a newline before "For example"? I'd prefer sticking with single spaces everywhere.